Added Flagsmith datadog widget (#1793)
DataDog/integrations-extras: flagsmith/README.md

```javascript
flagsmith.init({
     datadogRum: {
        client: datadogRum,
        trackTraits: true,
    },
    ...
})
```

### Flagsmith Dashboard widget

1. On the [Flagsmith integration tile](https://app.datadoghq.com/integrations/flagsmith), make sure the integration is installed.

> 1. Make sure you are logged into Flagsmith with the account you want to see in Datadog.

Reviewer: ```suggestion
2. Make sure you are logged into Flagsmith with the account you want to see in Datadog.
```

> 1. In Datadog, navigate to an existing dashboard or create a new one.

Reviewer: ```suggestion
3. In Datadog, navigate to an existing dashboard or create a new one.
```

> 1. Press the **Add Widgets** button to expose the widget drawer.

Reviewer: ```suggestion
4. Press the **Add Widgets** button to expose the widget drawer.
```

> 1. Search for **Flagsmith** to find the Flagsmith widget in the **Apps** section of the widget drawer.

Reviewer: ```suggestion
5. Search for **Flagsmith** to find the Flagsmith widget in the **Apps** section of the widget drawer.
```

> 1. Select the **Flagsmith widget icon** to add it your your dashboard and open the **Flagsmith editor** modal. You can choose to add either the Flag or Audit log viewer widget.

Reviewer: ```suggestion
6. Select the **Flagsmith widget icon** to add it your your dashboard and open the **Flagsmith editor** modal. You can choose to add either the Flag or Audit log viewer widget.
```

> 1. Select the Flagsmith Organisation, Project and Environment you want to add to your dashboard.

Reviewer: ```suggestion
7. Select the Flagsmith Organization, Project and Environment you want to add to your dashboard.
```

> 1. Once selected, copy and paste the **Project ID** and **Environment ID** into Datadog.

Reviewer: ```suggestion
8. Once selected, copy and paste the **Project ID** and **Environment ID** into Datadog.
```

> 1. Select the page size and, optionally, a widget title and Flagsmith Tag to filter on.

Reviewer: ```suggestion
9. Select the page size and, optionally, a widget title and Flagsmith Tag to filter on.
```

> 1. Click **Save** to finish configuring the dashboard widget.

Reviewer: ```suggestion
10. Click **Save** to finish configuring the dashboard widget.
```
